.github/workflows/ci.yml

```diff
@@ -55,8 +55,11 @@ jobs:
         module_path=$work_path/lib
         sed -e "s#your_path#$module_path#g" tests/tairzset.tcl > redis/tests/unit/type/tairzset.tcl
         sed -i 's#unit/type/string#unit/type/tairzset#g' redis/tests/test_helper.tcl
+        sed -e "s#your_path#$module_path#g" tests/tairzset-6.0.0.tcl > redis/tests/unit/type/tairzset-6.0.0.tcl 
+        sed -i 's#unit/type/string#unit/type/tairzset-6.0.0#g' redis/tests/test_helper.tcl
```

Comment:
Do you check this replace is work ? You use the pattern `unit/type/string` replace twice in test_help.tcl file.

Reply:
Oh, I didn't figure out that. But in fact, it's not necessary to put the test unit name into `redis/tests/test_helper.tcl`. Just put the tcl file into `redis/tests/unit/type/` is ok.

Reply:
Yes, you are right. so u can just copy the `tairzset-6.0.0.tcl`  to `redis/tests/unit/type/ `.